framework/src/org/checkerframework/framework/type/AnnotatedTypeMirror.java

```diff
@@ -1512,9 +1509,6 @@ void setLowerBoundField(AnnotatedTypeMirror type) {
          * @return the lower bound field.
          */
         public AnnotatedTypeMirror getLowerBoundField() {
-            if (lowerBound != null) {
```

Comment:
Note: I don't like this approach but it was a compromise I made while in the thick of type variable changes and on a deadline.  I guess the alternative would be to go through and fix all cases where annotations are accessed directly and put them through addAnnotation